tests/src/test/spark330/scala/org/apache/spark/sql/rapids/GpuInSubqueryExecSuite.scala

```diff
@@ -65,24 +65,27 @@ class GpuInSubqueryExecSuite extends SparkQueryCompareTestSuite {
 
   private def buildCpuInSubqueryPlan(
       spark: SparkSession,
-      shouldBroadcast: Boolean): SparkPlan = {
+      shouldBroadcastOrDpp: Boolean): SparkPlan = {
```

Comment:
My preference is to minimize the change. I would leave the name as is, then we could have the whole patch a one-line change